tests/protractor/page-objects/forms/delivery-report.po.js

```diff
@@ -1218,21 +1188,27 @@ module.exports = {
 
   //note to CHW
   getNoteToCHW: () => {
-    return element(by.name('/delivery/group_note/g_chw_sms')).getText();
+    return element(by.css('textarea')).getAttribute('value');
```

Comment:
This seems like a very brittle way to reference a specific field, and `getAttribute('value')` is a surprising way to get the value of a textarea, when normally the value is not stored in an attribute.